tq/transfer_queue_test.go

```diff
@@ -31,6 +44,36 @@ func TestRetryCounterCanNotRetryAfterExceedingRetryCount(t *testing.T) {
 	assert.False(t, canRetry)
 }
 
+func TestRetryCounterDoesNotDelayFirstAttempt(t *testing.T) {
+	rc := newRetryCounter()
+	assert.Equal(t, time.Time{}, rc.ReadyTime("oid"))
+}
+
+func TestRetryCounterDelaysExponentially(t *testing.T) {
+	rc := newRetryCounter()
+	start := time.Now()
+
+	rc.Increment("oid")
+	ready1 := rc.ReadyTime("oid")
+	assert.GreaterOrEqual(t, ready1.Sub(start).Milliseconds(), int64(baseRetryDelayMs))
```

Comment:
It looks like this item and the below one are failing since they don't work on Go 1.12.  Could we maybe use `Nanoseconds` and multiply?